src/client/pages/OfferNew/Tabs/Underline.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import styled from '@emotion/styled'
+import { motion } from 'framer-motion'
+import { colorsV3 } from '@hedviginsurance/brand'
+import { MEDIUM_SCREEN_MEDIA_QUERY } from 'utils/mediaQueries'
+
+const Underline = styled(motion.div)`
+  box-shadow: 0px 2px 0px ${colorsV3.gray900};
```

Comment:
Did you get this to work with box-shadow? :D

Reply:
this one yes. but the Tab needs border-bottom :(